README.md

```diff
@@ -15,4 +15,4 @@ Package|Description
 ---|---
 cf-cli|The official Cloud Foundry CLI
 bosh-init|The official CLI to create and update the BOSH Director VM
-bosh-cli|The new golang BOSH cli (alpha)
+bosh-cli-alpha|The new golang BOSH cli (alpha), binary will be added as gosh to avoid conflicts with the old cli.
```

Comment:
Renamed the binary, as stated in my previous comment. This makes more sense for now in my opinion, since we haven't deprecated the old cli, and this is still "alpha".